Hymns/Home/LaunchRouterView.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+struct LaunchRouterView: View {
+    @State var showSplash: Bool = false
+
+    var body: some View {
+        Group { () -> AnyView in
+            if showSplash {
+                return SplashScreenView()
+                    .opacity(showSplash ? 1 : 0)
+                    .onAppear {
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 2.7) {
+                            self.showSplash = false
+                        }
+                }.eraseToAnyView()
+            } else {
+                return HomeContainerView().eraseToAnyView()
```

Comment:
So here you are showing the HomeContainer inside of the LaunchRouterView. This means that all future views will be nested inside of this LauncherView.

Is it not possible in SwiftUI to push and pop views programmatically?

Reply:
oops, I meant to post my comments here: 

> I came across this: https://stackoverflow.com/a/62225392/1907538. Do you think something like that is feasible?

> another possibility: https://stackoverflow.com/a/57274613/1907538
> Since we aren't using the back button in this case, maybe we can get away with it?

Reply:
I don't think that is really a problem because we don't declare a navigationView in launch router. So the root for our navigation stack should still be HomeContainer right?